exercises/hello-world/hello_test.go

```diff
@@ -3,12 +3,16 @@ package greeting
 import "testing"
 
 // Define a function HelloWorld(string) string.
-//
-// Also define a testVersion with a value that matches
-// the targetTestVersion here.
 
+// In any exercise solution using these tests you must also define a
```

Comment:
It seems a bit unclear to me (non-native speaker) what "these" is referring to here?

In `hello_world.go` the comments were explicit that it was referring to exercises of the Go track. Perhaps the same can be done here?